datasets/simulated.py

```diff
@@ -12,10 +12,10 @@ class Dataset(BaseDataset):
     # the cross product for each key in the dictionary.
     parameters = {
         'n_samples, n_features': [
-            (100, 5_000),
-            (100, 10_000)
+            (200, 500),
+            (500, 200)
         ],
-        'rho': [0, 0.6],
+        'rho': [0.6],
```

Comment:
not sure why change the parameters? to make default run quicker?

Reply:
yes but it does not belong to this PR, I'll revert